src/mbgl/style/layers/line_layer.cpp

```diff
@@ -431,5 +431,32 @@ TransitionOptions LineLayer::getLinePatternTransition() const {
     return impl().paint.template get<LinePattern>().options;
 }
 
+ColorRampPropertyValue LineLayer::getDefaultLineGradient() {
+    return { {} };
+}
+
+ColorRampPropertyValue LineLayer::getLineGradient() const {
+    return impl().paint.template get<LineGradient>().value;
+}
+
+void LineLayer::setLineGradient(ColorRampPropertyValue value) {
+    if (value == getLineGradient())
+        return;
```

Comment:
Braces

Reply:
This is a generated file, I could introduce braces generation to `scripts/generate-style-code.js` in a follow-up PR